server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -89,22 +89,30 @@ async function updateExistingSession(
         ) {
             // Previous session was in a different location. Move to current location.
             // Reset logOffset, ordererUrl, and historianUrl when moving session location.
-            Lumberjack.info(
-                `Reset logOffset, ordererUrl, and historianUrl when switching cluster.`,
-                lumberjackProperties,
-            );
+            Lumberjack.info("Moving session", {
+                ...lumberjackProperties,
+                isSessionSticky,
+                documentLastAccessTime: document.lastAccessTime,
+                dataTime: Date.now(),
```

Comment:
Why log dateTime? We already have eventTimeStamp, so this feels unnecessary. Am I missing something?

Reply:
Got it. In that case, is "data" a typo meant to be "date"?

Reply:
Thanks for catching it. :)

Reply:
@tianzhu007 I'm not sure I understand. I agree with @znewton in that we don't need to log a `dateTime` property. We already have a timestamp property in the `Lumber` type, used internally by `Lumberjack`:
https://github.com/microsoft/FluidFramework/blob/77d5f290bcc039a5221d10f3201ff5fc886d096b/server/routerlicious/packages/services-telemetry/src/lumber.ts#L32

If `isSessionSticky` calculates `Date.now() - document.lastAccessTime`, can we instead do one of the following:
1. Instead of adding a redundant `dateTime` property, maybe you can add a `sessionStickinessTime` property instead if  `document.lastAccessTime` is available to be used here?
2. Or just omit this redundant `dateTime` completely and instead use the one already in `Lumber`?